test/link_transformer_test.rb

```diff
@@ -39,42 +38,38 @@
+        The 3.0 release of the google-cloud-translate client is a significant upgrade
+        based on a [next-gen code generator](https://github.com/googleapis/gapic-generator-ruby),
+        and includes substantial interface changes. [Set up authentication](AUTHENTICATION.md),
+        [Set up authentication](AUTHENTICATION.md) [MIGRATION.md](MIGRATING.md)
       TEXT
 
       transformed_text = link_transformer.transform_links_in_text text
 
       yard_text = <<~TEXT
-      The 3.0 release of the google-cloud-translate client is a significant upgrade
-      based on a [next-gen code generator](https://github.com/googleapis/gapic-generator-ruby),
-      and includes substantial interface changes. {file:AUTHENTICATION.md Set up authentication},
-      {file:AUTHENTICATION.md Set up authentication} {file:MIGRATING.md MIGRATION.md}
+        The 3.0 release of the google-cloud-translate client is a significant upgrade
+        based on a [next-gen code generator](https://github.com/googleapis/gapic-generator-ruby),
+        and includes substantial interface changes. {file:AUTHENTICATION.md Set up authentication},
+        {file:AUTHENTICATION.md Set up authentication} {file:MIGRATING.md MIGRATION.md}
       TEXT
 
       expect(transformed_text).must_equal yard_text
     end
 
     it "tranforms all the instances of md links to yard links in text" do
       text = <<~TEXT
-      In order to use this library, you first need to go through the following steps:
-
-      1. [Select or create a Cloud Platform project.](https://console.cloud.google.com/project)
-      1. [Enable billing for your project.](https://cloud.google.com/billing/docs/how-to/modify-project#enable_billing_for_a_project)
-      1. [Enable the API.](https://console.cloud.google.com/apis/library/translate.googleapis.com)
-      1. [Set up authentication](AUTHENTICATION.md)
-      
-      ## Migrating from 2.x versions
-      
-      The 3.0 release of the google-cloud-translate client is a significant upgrade
-      based on a [next-gen code generator](https://github.com/googleapis/gapic-generator-ruby),
-      and includes substantial interface changes. Existing code written for earlier
-      versions of this library will likely require updates to use this version.
-      See the [MIGRATION.md](MIGRATING.md) document for more information.
+        In order to use this library, you first need to go through the following steps
+        1. [Select or create a Cloud Platform project.](https://console.cloud.google.com/project)
+        1. [Enable billing for your project.](https://cloud.google.com/billing/docs/how-to/modify-project#enable_billing_for_a_project)
+        1. [Enable the API.](https://console.cloud.google.com/apis/library/translate.googleapis.com)
+        1. [Set up authentication](AUTHENTICATION.md)
+        ## Migrating from 2.x versions
+        The 3.0 release of the google-cloud-translate client is a significant upgrade
```

Comment:
Any reason why you dropped the blank lines here?

Reply:
I don't think Rubocop was too happy with the blank lines here

Reply:
We  should be able to include blank lines in a multi-line string like this. Maybe make sure the lines are truly empty (and don't have any spaces in them for example).

Reply:
I was able to include the blank lines between the bullets in the list. But in other parts, I would hit a trailing whitespace error?